### DISCUSSION TITLE
Not in Application folder error after updating/installing Xcode

### DISCUSSION THREAD

**Discussion by amirsaam** [PlayCover/PlayCover#336] [General]
After updating Xcode (I think at least major updates like the recent 14) or installing it, you probably get a "Not in Application folder" error from PlayCover if you didn't open Xcode and accept Apple ToS.
For resolving this you need to open Xcode and accept Apple ToS then the error will be gone.

You can also run `sudo xcodebuild -license accept` in terminal to fix the error.

By the way, if you don't have Xcode and you got this error any way, you should consider updating it's either two way:
- You don't have Xcode Command Line Tools, consider installing it by using `xcode-select --install` command in your Terminal.
- You didn't update your Xcode Command Line Tools to the latest version through Mac's System Preferences > Software Update

If you somehow still getting this error after doing the steps above try this command in your Terminal: `sudo xcrun cc`.

**Comment by zhugunick** [comment]
How to solve the crash when using lldb to open genshin?
